gestalt-inject-java/src/main/java/org/terasology/gestalt/annotation/processing/AnnotationTypeWriter.java

```diff
@@ -12,22 +12,38 @@
 import java.util.HashSet;
 import java.util.Map;
 
+/**
+ * Class to write annotation types.
+ */
 public class AnnotationTypeWriter {
     private static final String META_INF = "META-INF/annotations";
 
     private final Filer filer;
     private final Map<String, HashSet<String>> results = new HashMap<>();
     private final Map<String, FileObject> files = new HashMap<>();
 
+    /**
+     * Creates a AnnotationTypeWriter to write an annotation.
```

Comment:
```suggestion
     * Creates an AnnotationTypeWriter.
```

Short and not very useful but this constructor doesn't do much anyway. This is probably enough.